R/bcdc_search.R

```diff
@@ -120,9 +120,14 @@ bcdc_list_organizations <- function() bcdc_search_facets("organization")
 bcdc_list_organization_records <- function(organization) {
   if(!has_internet()) stop("No access to internet", call. = FALSE) # nocov
 
-  cli <- bcdc_catalogue_client("action/organization_show")
+  query <- paste0("organization:", organization)
```

Comment:
Just a nit, but can this be specified as `list(organization = organization)` to match the way we specify the queries in other calls? Or does that not work here?